docs/01-app/03-api-reference/04-functions/revalidatePath.mdx

```diff
@@ -30,7 +30,7 @@ revalidatePath(path: string, type?: 'page' | 'layout'): void;
 
 ```ts
 import { revalidatePath } from 'next/cache'
-revalidatePath('/blog/post-1')
+revalidatePath('/blog/post-1/')
```

Comment:
Can you provide a deployment/repro where you are seeing this? We don't add the trailing slash to the tags generated so shouldn't be the case when calling revalidatePath either. 

x-ref: https://github.com/vercel/next.js/blob/a362b340f23798663ce17db49c051ca48ddc91ca/packages/next/src/server/lib/implicit-tags.ts#L43